isisd/isis_pdu.c

```diff
@@ -151,6 +152,49 @@ static int process_p2p_hello(struct iih_info *iih)
 	struct isis_adjacency *adj = iih->circuit->u.p2p.neighbor;
 	/* If an adjacency exists, check it is with the source of the hello
 	 * packets */
+	if ((((iih->circuit->area->is_type == IS_LEVEL_1) &&
+				(iih->circuit->is_type_config == IS_LEVEL_1_AND_2 ||
```

Comment:
The indentation needs to be fixed here.  Please use clang-format

Reply:
For some reason clang-format -i isisd/isis_pdu.c formats some other lines which I didn't add/change in commit. What could be the matter?